src/infra/dex/balancer/dto.rs

```diff
@@ -1,58 +1,208 @@
 use {
     crate::{
-        domain::{auction, dex, order},
+        domain::{dex, eth, order},
+        infra::dex::balancer::Error,
         util::serialize,
     },
+    bigdecimal::BigDecimal,
     ethereum_types::{H160, H256, U256},
+    number::conversions::{big_decimal_to_u256, u256_to_big_decimal},
     serde::{Deserialize, Serialize},
     serde_with::serde_as,
 };
 
-#[derive(Debug, Serialize)]
+pub mod get_swap_paths_query {
+    /// Get swap quote from the SOR v2 for the V2 vault.
```

Comment:
Isn't this v3 now?

Reply:
This comes from the API docs. That is what the balancer team says about it: https://cowservices.slack.com/archives/C035TDPHQAJ/p1717783431010809?thread_ts=1717602597.107859&cid=C035TDPHQAJ